contrib/src/main/java/com/datatorrent/contrib/avro/AvroRecordHelper.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.datatorrent.contrib.avro;
+
+import java.text.ParseException;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.apache.avro.Schema;
+import org.apache.avro.Schema.Type;
+
+/**
+ * This is an utility class for reading Avro converted records
+ */
+public class AvroRecordHelper
```

Comment:
This can be used by other classes for resolving avro schema.Its expected to be used as a helper to get values associated with all Avro types both primitive & non-primitive.A user can use it directly as a part of this operator to resolve the same.